### DISCUSSION TITLE
是否有pip包远程连接GraphScope

### DISCUSSION THREAD

**Discussion by hzfei** [alibaba/GraphScope#1518] [General]
今天刚看GraphScope，跟neo4j区别很大，一时间还看不懂GraphScope到底是个啥。有几个问题：
1.GraphScope是否可以像neo4j那样存储数据？
2.GraphScope是否有类似py2neo那样的包远程访问neo4j?

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! And a maintainer will get back to you shortly!
Please feel free to contact us on [DingTalk](https://h5.dingtalk.com/circle/healthCheckin.html?dtaction=os&corpId=dingb4b76e0fa9ccd8d516fd0f0acdf19f55&68e71190=0fe34ec0&cbdbhh=qwertyuiop), WeChat account(graphscope) or [Slack](http://slack.graphscope.io/). We are happy to answer your questions responsively.

**Comment by lidongze0629** [comment]
@hzfei [GraphScope](https://graphscope.io/) 是一款**一站式的大规模**图计算系统，其可以用于完成图分析(如 pagerank)、图交互式查询(gremlin)、和图学习(learning) 等计算任务，你可以通过 [文档](https://graphscope.io/docs/zh/index.html) 或 [PlayGround](https://try.graphscope.app) 来进一步了解、试用 GraphScope。

**1. GraphScope是否可以像neo4j那样存储数据？**

[GraphScope Store](https://github.com/alibaba/GraphScope/tree/main/research/graph_store) 可以像 neo4j 一样用于图数据的存储

**2. GraphScope是否有类似py2neo那样的包远程访问neo4j?**

目前 GraphScope 无法远程访问 neo4j，数据的载入支持 HDFS、OSS、Local 数据源

**Comment by hzfei** [comment]
谢谢，网上文档太乱了。



---原始邮件---
发件人: "DongZe ***@***.***&gt;
发送时间: 2022年5月6日(周五) 晚上11:17
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [alibaba/GraphScope] 是否有pip包远程连接GraphScope (Issue #1514)




 
@hzfei GraphScope 是一款一站式的大规模图计算系统，其可以用于完成图分析(如 pagerank)、图交互式查询(gremlin)、和图学习(learning) 等计算任务，你可以通过 文档 或 PlayGround 来进一步了解、试用 GraphScope。
 
1. GraphScope是否可以像neo4j那样存储数据？
 
GraphScope Store 可以像 neo4j 一样用于图数据的存储
 
2. GraphScope是否有类似py2neo那样的包远程访问neo4j?
 
目前 GraphScope 无法远程访问 neo4j，数据的载入支持 HDFS、OSS、Local 数据源

**Comment by lidongze0629** [comment]
欢迎对文档提出任何建议哈，也建议你可以从 [Get Started](https://github.com/alibaba/GraphScope/blob/main/README.md) 和 [PlayGround](https://try.graphscope.app) 上手，来试用 GraphScope
